### PR TITLE
Prevent exception if their is no else block.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Previously would fail with `Null Reference Exception` , if the `ifBlock` didn't have and `elseBlock`
```vbnet
If condition Then
  Console.WriteLine("Statement")
End If
```

### CODE REVIEW

File: src/Features/VisualBasic/Portable/InvertIf/VisualBasicInvertIfCodeRefactoringProvider.MultiLine.vb [hunk 1]
```diff
@@ -67,7 +67,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InvertIf
         End Function
 
         Private Shared Function HasComment(elseBlock As ElseBlockSyntax) As Boolean
-            Return elseBlock.GetLeadingTrivia().Any(Function(trivia) trivia.IsKind(SyntaxKind.CommentTrivia))
+        Return elseBlock IsNot Nothing AndAlso elseBlock.GetLeadingTrivia().Any(Function(trivia) trivia.IsKind(SyntaxKind.CommentTrivia))
```
Comment (ryzngard) [inline src/Features/VisualBasic/Portable/InvertIf/VisualBasicInvertIfCodeRefactoringProvider.MultiLine.vb:70]:
```suggestion
            Return elseBlock IsNot Nothing AndAlso elseBlock.GetLeadingTrivia().Any(Function(trivia) trivia.IsKind(SyntaxKind.CommentTrivia))
```

**Review by ryzngard — APPROVED** [review]

**Review by ryzngard — CHANGES_REQUESTED** [review]
Can you add a test for this case?

**Comment by AdamSpeight2008** [comment]
> Can you add a test for this case?

I don't know where the unit tests are? For this area of the Roslyn repo.

**Comment by ryzngard** [comment]
> > Can you add a test for this case?
> 
> I don't know where the unit tests are? For this area of the Roslyn repo.

It doesn't look like there are any :( We have csharp but no VB tests. https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/CSharpTest/InvertIf/InvertIfTests.cs 

Filed #55677 to add tests

**Review by ryzngard — APPROVED** [review]

**Comment by AdamSpeight2008** [comment]
Example that triggera the exception in current version of code.
```vb,net
Imports System

Module Program
  Sub Main(args As String())
    Dim condition As Boolean = False
    If condition Then ' Trigger Code Action on this line.
      Console.WriteLine("Hello World!")
    End If
  End Sub
End Module

```

**Comment by CyrusNajmabadi** [comment]
We shoudl still add tests here.  @AdamSpeight2008 if you'd like, we can do that though.  Please let us know.  @ryzngard can you help with that if Adam doesn't have time?

**Comment by ryzngard** [comment]
@CyrusNajmabadi yea, I filed an issue and assigned to myself. We definitely need coverage here and I should be able to get tests going this afternoon.

**Comment by AdamSpeight2008** [comment]
@ryzngard If you can add the unit test for this one, I'll then help adding additional coverage and examples.

**Comment by AdamSpeight2008** [comment]
@ryzngard Add an unit test for this case.

**Comment by ryzngard** [comment]
> @ryzngard Add an unit test for this case.

I'll add one this weekend

**Comment by ryzngard** [comment]
Ah, I see you did :)

**Pull request merged**